### PR TITLE
feat: Add banner and infostripe for PCSX

### PR DESCRIPTION
**Pull request by chef-ryan**
Add new banner and info stripe for PCS X
see:
https://linear.app/pancakeswap/issue/PAN-3937/add-new-banner-and-info-stripe-for-pcs-x

---

## PR-Codex overview
This PR introduces the `PCSXBanner` component to the application, enhancing the user interface with new banners and features. It also refines the phishing banner logic and updates localization strings.

### Detailed summary
- Added `PCSXBanner` component with promotional content.
- Integrated `Step3` component into the `PhishingWarningBanner`.
- Updated `useMultipleBannerConfig` to include `PCSXBanner`.
- Modified phishing banner logic for better condition handling.
- Expanded localization strings in `translations.json`.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by ChefJerry — APPROVED** [review]

File: packages/utils/user/phishingBanner.ts [hunk 1]
```diff
@@ -8,8 +8,8 @@ const hidePhishingBannerAtom = atom(
   (get) => {
     const now = dayjs()
     const last = dayjs(get(phishingBannerAtom)).add(1, 'day')
-    const notPreview = process.env.NEXT_PUBLIC_VERCEL_ENV !== 'preview'
-    return last ? now.unix() > last.unix() && notPreview : notPreview
+    // const notPreview = process.env.NEXT_PUBLIC_VERCEL_ENV !== 'preview'
```
Comment (thechefpenguin) [inline packages/utils/user/phishingBanner.ts:11]: Do we want to revert this or is this an intended change?

File: apps/web/src/components/PhishingWarningBanner/Step3.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import { useTranslation } from '@pancakeswap/localization'
+import { ArrowForwardIcon, Box, Flex, Link, Text } from '@pancakeswap/uikit'
+import { VerticalDivider } from '@pancakeswap/widgets-internal'
+
+export const Step3 = () => {
+  const { t } = useTranslation()
+
+  // const PlayNowAction = (
```
Comment (thechefpenguin) [inline apps/web/src/components/PhishingWarningBanner/Step3.tsx:8]: Can remove commented part if not needed

File: packages/utils/user/phishingBanner.ts (hunk 1 above)
Comment (chefjackson) [reply packages/utils/user/phishingBanner.ts:11]: don't have a strong opinion on this. Both looks fine to me

Comment (thechefpenguin) [reply packages/utils/user/phishingBanner.ts:11]: Okay

File: apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx [hunk 3]
```diff
@@ -40,6 +41,10 @@ export const useMultipleBannerConfig = () => {
 
   return useMemo(() => {
     const NO_SHUFFLE_BANNERS: IBannerConfig[] = [
+      {
+        shouldRender: true,
+        banner: <PCSXBanner />,
+      },
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx:47]: Move this under `banner: <UserBanner />`

File: apps/web/src/views/Home/components/Banners/PCSXBanner.tsx [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+    <StyledBannerDesc>
+      {t('0 Trading Fees, Gasless Swaps, MEV Protection, and a Wide Range of Tokens — ALL on PancakeSwapX')}
+    </StyledBannerDesc>
+  )
+}
+
+export const PCSXBanner = () => {
+  const { t } = useTranslation()
+  const { isMobile, isSm, isDesktop } = useMatchBreakpoints()
+  console.log({ isMobile, isSm, isDesktop })
```
Comment (thechefpenguin) [inline apps/web/src/views/Home/components/Banners/PCSXBanner.tsx:94]: Can remove console.log

File: apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx (hunk 3 above)
Comment (chef-ryan) [reply apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx]: done

File: apps/web/src/views/Home/components/Banners/PCSXBanner.tsx (hunk 4 above)
Comment (chef-ryan) [reply apps/web/src/views/Home/components/Banners/PCSXBanner.tsx]: done

File: apps/web/src/components/PhishingWarningBanner/Step3.tsx (hunk 2 above)
Comment (chef-ryan) [reply apps/web/src/components/PhishingWarningBanner/Step3.tsx]: done

File: packages/utils/user/phishingBanner.ts (hunk 1 above)
Comment (chef-ryan) [reply packages/utils/user/phishingBanner.ts]: The preview environment doesn't display the info stripe, which may not be a good choice. Let's remove it for now and see.

**Review by ChefMomota — APPROVED** [review]

**Comment by ChefMomota** [comment]
Mobile issue
Screenshot 2024-10-17 at 11 21 53 AM

**Review by ChefMomota — APPROVED** [review]

**Pull request merged**